theSrc/scripts/LabeledScatter.js

```diff
@@ -188,6 +191,25 @@ if (
 
     return this.resizeDelayPromise
   }
+
+  addMarkerClickHandler () {
+    const el = d3.select(this.rootElement).select('.nsewdrag');
+    el[0][0].onclick = ((e) => {
```

Comment:
Maybe just add a comment here about the why the euclidean distance is needed? I would have thought if you were close enough to click the nsewdrag element that you were pretty close already

Reply:
I've added a comment in 1cb56c6003ddfd22b57282eeecea024d39682c7c.

nsewdrag is actually a rectangle covering the entire plot area. The euclidean distance is used so that we only register clicks on the marker itself, not around it. I've added the explanatory variable `is_marker_clicked_on` to hopefully make this clearer.

Reply:
Thanks, that explanation about nsewdrag was helpful